app/src/main/java/org/fossasia/phimpme/editor/fragment/AddTextFragment.java

```diff
@@ -216,10 +216,14 @@ public void backToMain() {
     public void onShow() {
         activity.changeMode(EditImageActivity.MODE_TEXT);
         activity.mainImage.setImageBitmap(activity.mainBitmap);
+        activity.addTextFragment.getmTextStickerView().mainImage = activity.mainImage;
+        activity.addTextFragment.getmTextStickerView().mainBitmap = activity.mainBitmap;
         mTextStickerView.setVisibility(View.VISIBLE);
         mInputText.clearFocus();
     }
 
+    public TextStickerView getmTextStickerView(){return mTextStickerView;}
```

Comment:
Can this be made static?

Reply:
Why do u think so?

Reply:
Making the function static would lead to conversion of other variables to static as well . Also the same approach (no static function) has been used in case when the paint and the text was going out of bounds as well.